autonomy/deploy/chain.py

```diff
@@ -33,7 +33,7 @@
 SERVICE_REGISTRY_ABI = (
     DATA_DIR / ABIS_DIR / "service_registry" / "service_registry.json"
 )
-DEFAULT_STAGING_CHAIN = "https://chain.staging.autonolas.tech"
+DEFAULT_STAGING_CHAIN = "http://localhost:8545"
```

Comment:
Can you also update the relevant docs on how to use local registry deployment or this won't work. The script to run the local registry deployment is `scripts/run_service_registry_locally.py`

Reply:
Is there a way to run this script from `autonomy` ? Otherwise we make the users life way too difficult IMO. They would have to go and get the script from the open-autonomy github.

Reply:
Alternatively, we can make the default chain to be goerli. And give an explanation on how to run the localnet.

Reply:
Let;s open an Trello ticket for making the script above part of autonomy.